addons/edi/models/edi_record.py

```diff
@@ -99,6 +99,10 @@ class EdiRecord(models.AbstractModel):
     BATCH_UPDATE = property(attrgetter("BATCH_SIZE"))
     """Batch size for updating existing records"""
 
+    # TODO
```

Comment:
remove (if all sorted)

Reply:
I'll remove because they aren't really in scope for this PR.